### PR TITLE
MiniMap bugfixes

### PR DESCRIPTION
**Pull request by jorenham**
- Fixed outlier correction
- Fixed memory overflow for large amount of buckets

### CODE REVIEW

**Comment by albert-smit** [comment]
looks good

**Comment by joswinter** [comment]
looks good

File: lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/model/MiniMap.java [hunk 1]
```diff
@@ -35,6 +43,7 @@
      */
     public MiniMap(final BucketCache bucketCache) {
         sumBuckets(bucketCache.getBuckets());
+        System.out.println(scores);
```
Comment (rutgervandenberg) [inline lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/model/MiniMap.java]: println!@!@!@

File: lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/model/MiniMap.java [hunk 2]
```diff
@@ -69,12 +98,49 @@ private Color colorFromScore(final double score) {
...
+        Collections.sort(sorted);
+        return sorted.get(sorted.size() / 2);
+    }
+
+    /**
+     * @param collection
+     *            the collection to take the max of
+     * @return the biggest double in the collection
+     */
+    private static double max(final Collection<Double> collection) {
```
Comment (rutgervandenberg) [inline lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/model/MiniMap.java]: Collections.max(Collection) does the same thing.

**Comment by alangerak** [comment]
Looks good

**Pull request merged**